src/cow-react/common/services/getQuoteCurrency/index.ts

```diff
@@ -24,11 +25,13 @@ const STABLE_COINS: { [key in SupportedChainId]: string[] } = {
  * 2. Otherwise, take the token with the smallest amount as quote (for 0.0005 WETH -> 3000 COW, WETH is quote)
  */
 export function getQuoteCurrency(
-  chainId: SupportedChainId | undefined,
+  _chainId: SupportedChainId | undefined,
   inputCurrencyAmount: CurrencyAmount<Currency> | null,
   outputCurrencyAmount: CurrencyAmount<Currency> | null
 ): Currency | null {
-  if (!chainId || !inputCurrencyAmount || !outputCurrencyAmount) return null
+  const chainId = supportedChainId(_chainId)
+
+  if (chainId || !inputCurrencyAmount || !outputCurrencyAmount) return null
```

Comment:
Shouldn't it be this?

```suggestion
  if (!chainId || !inputCurrencyAmount || !outputCurrencyAmount) return null
```